graf2d/gpad/inc/TRatioPlot.h

```diff
@@ -21,21 +21,22 @@
 //                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
-#include "TH1.h"
-#include "TPad.h"
-#include "TGraphAsymmErrors.h"
-#include "TGraphErrors.h"
-#include "TGraph.h"
-#include "TAxis.h"
-#include "TGaxis.h"
-#include "TH1F.h"
-#include "TFitResultPtr.h"
-#include "THStack.h"
-
+class TObject;
+class TH1;
+class TPad;
+class TGraphAsymmErrors;
+class TGraphErrors;
+class TGraph;
+class TAxis;
+class TGaxis;
+class TLine;
+class TFitResultPtr;
+class TFitResult;
+class THStack;
 class TBrowser;
 class TFileMergeInfo;
 
-class TRatioPlot : public TPad {
+class TRatioPlot : public TObject {
```

Comment:
If you inherit from TObject (and you should) then we need to `#include` TObject.h. If you *source* file starts with `#include "TRatioPlot.h"` then it should have told you - which is why that should be the first `#include` :-)